packages/cardano-services/test/Program/utils.test.ts

```diff
@@ -495,17 +496,20 @@ describe('Service dependency abstractions', () => {
       await listenPromise(mockServer, connection);
       await ogmiosServerReady(connection);
 
-      txSubmitWorker = await loadTxWorker({
-        options: {
-          cacheTtl: 10_000,
-          loggerMinSeverity: 'error',
-          ogmiosSrvServiceName: process.env.OGMIOS_SRV_SERVICE_NAME,
-          parallel: true,
-          rabbitmqSrvServiceName: process.env.RABBITMQ_SRV_SERVICE_NAME,
-          serviceDiscoveryBackoffFactor: 1.1,
-          serviceDiscoveryTimeout: 1000
-        }
-      });
+      txSubmitWorker = await loadTxWorker(
+        {
+          options: {
+            cacheTtl: 10_000,
+            loggerMinSeverity: 'error',
```

Comment:
```suggestion
```
This is now not needed since you're passing the `dummyLogger`

Reply:
6f3d639c8 should solve this